docs/tutorial/getting-started.rst

```diff
@@ -64,7 +63,13 @@ Next, you need to initialise DSS on top of MicroK8s and prepare MLflow:
 
    The initialisation might take a few minutes to complete.
    While the process is in progress, you will see the following message:
-   ``[INFO] Waiting for deployment my-tensorflow-notebook in namespace dss to be ready...``
+   ``[INFO] Waiting for deployment mlflow in namespace dss to be ready...``
```

Comment:
Got this feedback from a user, before it was wrong. First time you initialize, you see `mlflow`. Once you create a notebook, then `Waiting for deployment <your_notebook>` appears.

Reply:
Ah good point, it should be mlflow initially